docs/tutorial/VueJS/step4.md

```diff
@@ -110,15 +110,20 @@ Once again, the Sugar-Web environment could help us. When an activity is launche
 
 The good part is, `SugarJournal` handles this for you! It provides some events which you can handle however you want to. The events are:
 1. `journal-new-instance`: Emitted when the activity is a new instance.
-2. `journal-data-loaded`: Emitted when the activity is an existing instance and if there is some data stored in the journal. It returns two parameters: data and metadata.
-3. `journal-load-error`: Emitted when the activity is an existing instance and if some error occurred while loading data.
+2. `journal-shared-instance`: Emitted when the activity is a shared instance.
```

Comment:
I'm not sure it's pertinent to talk about it now because multi-user handling doesn't occur before step 6